src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/55.diff.sql

```diff
@@ -18,6 +15,8 @@ BEGIN
     ContentLength
     ) WITH (DATA_COMPRESSION = PAGE, ONLINE = ON)
 END
+
+BEGIN TRANSACTION
```

Comment:
Is this creating the index correctly? the GO statement can be after END and you can remove after Begin Transaction.